docs/source/_redirects

```diff
@@ -4,3 +4,4 @@
 /managed-federation/advanced-topics/ /docs/federation/managed-federation/overview/
 /managed-federation/monitoring/ /docs/federation/performance/monitoring
 /docs/federation/api/apollo-federation/ /docs/federation/api/apollo-subgraph/
+/workbench/using-federation-2/ /docs/federation/v1/workbench/overview/
```

Comment:
Those fed2 workbench things get redirected?  Or are they just folded into the new docs?